code-challenges.js

```diff
@@ -11,53 +12,104 @@ var num2 = 0
 
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
-
-
-
-
-
+const divideByThree = (num) => {
+    if (num % 3 === 0) {
+       return `${num} is divisible by three`
+    } 
+    else {return `${num} is not divisible by three`}
+}
```

Comment:
great! your last return doesn't need to be wrapped in `{ }` but otherwise this looks great!